docs/lua/guides/creating-mod.md

```
@@ -0,0 +1,71 @@
+Before continuing, if you haven't, I would highly advise reading up on the [Visual Studio Code setup guide](vs-code-setup.md) as it will provide autocomplete and docs when writing mods.
+
+For a file mod, simply open up the file in Visual Studio Code. For a folder mod, you can open the project in Visual Studio Code by dragging your folder into the app. Then open up `main.lua`.
+
+There are a few fields that exist for mods. Fields can be set at the very top of a file with comments and a newline for each option. Here is a list of options, as well as an example on what multiple options look like:
+
+| Category | Notes |
+| -------- | ----- |
+| `name` | The name of the mod. This name is the name used in game |
+| `description` | A description of the mod. Appears when hovering over a mod to turn it on/off |
+| `incompatible` | A list of tags to be marked as incompatible. This is an opt-in system. If 2 mods contain an equivalent incompatible tag, then they both won't be able to be enabled together. Tags are separated with a space character, or a whitespace.
+| `category` | The category of a mod. Leaving this empty will have the mod be in the `misc` category. The list of categories can be found in game by going to Host, Mods, then looping through the Categories select box. Note that categories are all lowercase, they are also only in English. So for category Utility, you want to put in your mod `-- category: utility`
```

markdownlint-cli2 (0.23.0)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)